lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -242,11 +252,7 @@ void StagesSubject::stageChanged(
                     Ufe::Transform3d::notify(ufePath);
                 }
             }
-#ifdef UFE_V2_FEATURES_AVAILABLE
-            if (!inAttributeChangedNotificationGuard()) {
-                sendValueChanged(ufePath, changedPath.GetNameToken());
-            }
-#endif
+            UFE_V2(valueChanged(ufePath, changedPath.GetNameToken());)
```

Comment:
Use the helper func which checks the notif guard. The previous code would also not add this value changed to the deferred notifs if we were in a guard.